django_app/redbox_app/templates/macros/chat-macros.html

```diff
@@ -54,7 +58,7 @@ <h3 class="iai-chat-bubble__sources-heading govuk-heading-s govuk-!-margin-botto
 </div>
 {% set seen_files = [] %}
 {% for file in message.selected_files.all() %}
-  {% if file not in seen_files %}
+  {% if file.id not in seen_files %}
```

Comment:
but the file.id differs dependent on the reference being appending at the end, this should be done on a different key that doesnt change